pkgs/development/python-modules/genson/default.nix

```diff
@@ -0,0 +1,49 @@
+{
+  buildPythonPackage,
+  coverage,
+  fetchFromGitHub,
+  jsonschema,
+  lib,
+  pytestCheckHook,
+  setuptools-scm,
+  wheel,
+}:
+
+buildPythonPackage rec {
+  pname = "genson";
+  version = "1.3.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "wolverdude";
+    repo = "GenSON";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-Bb2PRuZuj/yotb78MbLgVwi4Fz7hbnXJmoXTe4kg43k=";
+  };
+
+  build-system = [
+    setuptools-scm
+    wheel
+  ];
+
+  dependencies = [
+    coverage
+    jsonschema
+  ];
+
+  nativeCheckInputs = [ pytestCheckHook ];
+
+  # Currently it's not possible to run the tests
+  # ModuleNotFoundError: No module named 'genson.schema'
+  doCheck = false;
```

Comment:
`schema` exists [here](https://github.com/wolverdude/GenSON/tree/v1.3.0/genson), so if it is not found, it is probably not packaged correctly.